### ISSUE TITLE
Translation of "Terms of Use"

### ISSUE THREAD

**Issue by hauschke** [VIVO-DE/Vitro-languages#2]
termsOfUse_de_DE-ftl

Critical evaluation of the terms before translation is needed. Do these terms apply to Germany?

**Comment by grahamtriggs** [comment]
Good point, although that would mean the terms should be changed for all languages that the site will be displayed in. But possibly only for sites hosted in Germany.

Or are there possible modifications that can be suitable for other regions too?

**Comment by dofeldsc** [comment]
[TermsofUse-VIVO-Project-de_DE.pdf](https://github.com/VIVO-DE/Vitro-languages/files/4869103/TermsofUse-VIVO-Project-de_DE.pdf)
This is a initial translation as a basis for discussion.
